__tests__/app.test.js

```diff
@@ -0,0 +1,31 @@
+const app = require('../app');
+const request = require('supertest');
+const db = require('../db/connection');
+const seed = require('../db/seeds/seed');
+const data = require('../db/data/test-data');
+
+beforeEach(() => seed(data));
+afterAll(() => db.end());
+
+describe('App Tests', () => {
+    describe('GET tests', () => {
+        test('200: returns status code 200 upon successful GET request', () => {
+            return request(app)
+            .get('/api/topics')
+            .expect(200);
+        })
+        test('200: returns all topics with certain properties upon successful request', () => {
+            return request(app)
+            .get('/api/topics')
+            .then(({body}) => {
+                body.forEach(topic => {
+                    expect(topic).toHaveProperty('slug');
+                    expect(topic).toHaveProperty('description');
+                })
+            })
+        })
+
+    })
+    describe('Error handling tests', () => {
+    })
+})
```

Comment:
You could add a test here for '/api/banana' for example as a 404, however this will be handled by express so it's not strictly necessary to have error handlers at this point.